projects/smoldot-extension/src/utils/types.tsx

```diff
@@ -1 +1,20 @@
-export type NetworkTypes = 'kusama' | 'polkadot' | 'westend' | 'kulupu'
\ No newline at end of file
+export type NetworkTypes = 'kusama' | 'polkadot' | 'westend' | 'kulupu'
+export type NetworkStatus =  'connected' | 'ready' | 'disconnecting' | 'disconnected';
+
+export interface TabInterface {
+    tabId: number;
+    url: string;
+    uApps: uApp[];
+}
+export type uApp = {
+    networks: Networks[];
+    name: string;
+    enabled: boolean;
+}
+
+export type Networks = {
+    name: NetworkTypes;
+    status: NetworkStatus; 
+}
+
+export type NetworkCtx = TabInterface[];
```

Comment:
my obsessive–compulsive disorder goes brr when no new lines in the end of files (it's not you, it's me)

Reply:
Its you - I must at some point setup an eslinter that can be used for all projects so there is consistency between react.node projects